nixos/modules/services/networking/hostapd.nix

```diff
@@ -3,6 +3,7 @@
 # TODO:
 #
 # asserts
+#   ensure interface name is set
 #   ensure that the nl80211 module is loaded/compiled in the kernel
 #   wpa_supplicant and hostapd on the same wireless interface doesn't make any sense
```

Comment:
probably the same applies for this one as well - In `networking.supplicant.<name>`, `<name>` can be separated by spaces, so matching automatically would be a bit more complicated.